setup.py

```diff
@@ -64,4 +64,5 @@ def readfile(filename):
     ],
     python_requires=">=3.6",
     ext_modules=ext_modules,
+    package_data={"xxhash": ["py.typed"]},
```

Comment:
Do we also need to include __init__.pyi?

Reply:
Oh, I think so.